lets_meet/.env

```diff
@@ -4,3 +4,5 @@ EMAIL_PORT='587'
 EMAIL_USER='letsmeetsquad@gmail.com'
 EMAIL_PASSWORD='irvrjnyzfqcyxntj'
 EMAIL_TLS=True
+AUTH_ID="370195302414-q0sob5lr972rrrkv55qultt61oard9nu.apps.googleusercontent.com"
+AUTH_SECRET="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"
```

Comment:
Please add blank line in the end of the file

Reply:
done